### DISCUSSION TITLE
Question about download folders

### DISCUSSION THREAD

**Discussion by ChillyP** [UltimaHoarder/UltimaScraper#834] [General]
A couple of accounts I sub to have thousands of posts so I manually organise the images etc within the downloaded folder by year to make it easier to navigate in windows file explorer.

For example
Onlyfans/username/posts/free/Images/2021, 2020, 2019 etc etc.

In recent commits the script is taking the content from the manually created directories and putting them back into the root directory (i.e Images) and then deleting the empty directories, but it never use to do this?

This is my config file

  "supported": {
    "onlyfans": {
      "settings": {
        "auto_profile_choice": "0",
        "auto_scrape_names": true,
        "auto_choice": "a",
        "auto_scrape_apis": true,
        "jobs": {
          "scrape_names": true,
          "scrape_paid_content": false
        },
        "download_directories": [
          "D:/"
        ],
        "file_directory_format": "{site_name}\\{username}\\{api_type}\\{value}\\{media_type}",
        "filename_format": "{date}_{filename}.{ext}",
        "metadata_directories": [
          "D:/"
        ],
        "metadata_directory_format": "{site_name}\\{username}\\Metadata",
        "delete_legacy_metadata": false,
        "text_length": 255,
        "video_quality": "source",
        "overwrite_files": false,
        "date_format": "%d-%m-%Y",
        "ignored_keywords": [],
        "ignore_type": "",
        "blacklist_name": "blacklist",
        "webhook": true

Not an issue exactly but something I've seen with latest commits that the script didn't seem to do before, is there a change in my config file I could make that would allow keep the files in the folders I put them in without moving back?

**Comment by Nick05491** [comment]
Potential explaination here: https://github.com/DIGITALCRIMINAL/OnlyFans/discussions/1644
